File: 206_Reverse_Linked_List.js
var reverseList = function(head) {
    if (!head){
        return head
    }
    let first = head;
    let second = first.next;
    while(second){
        let temp = second.next;
        second.next = first;
        first = second;
        second = temp;
    }
    head.next = null
    head = first
    return head
};

File: Set1_1000/206_Reverse_Linked_List.py
class Solution:
    def reverseList(self, head: ListNode) -> ListNode:
        if not head:
            return head
        first = head
        second = first.next
        while(second):
            temp = second.next
            second.next = first
            first = second
            second = temp
            
        head.next = None
        head = first
        return head
